chronicnetwork/chain.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "Chronic Network",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Chronic",
+    "chain_id": "morocco-1",
+    "bech32_prefix": "chronic",
```

Comment:
Please register this prefix to https://github.com/satoshilabs/slips/blob/master/slip-0173.md